libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat

```diff
@@ -300,6 +300,11 @@ SPIDEV icm42688  SPI2 DEVID1  IMU2_CS MODE3  2*MHZ  8*MHZ
 SPIDEV icm42688_ext  SPI3 DEVID1  IMU3_CS MODE3  2*MHZ  8*MHZ
 #SPIDEV adis16507  SPI6 DEVID1  EXT1_CS MODE3  1*MHZ  2*MHZ
 
+# REV5
+SPIDEV iim42653  SPI1 DEVID2  IMU1_CS MODE3  2*MHZ  8*MHZ
```

Comment:
this PR won't do anything, as the iim42653 tag is not referenced anywhere. That tag is a string that needs to be referenced in an IMU line